src/modules/activities.js

```diff
@@ -0,0 +1,201 @@
+/* eslint-disable lines-between-class-members */
+/* eslint-disable no-plusplus */
+
+export default class ActivitiesManager {
+  #activitiesArr = [];
+  #createTask = (description, completed, number) => ({ description, completed, number });
+  #taskContainer;
+  #storage;
+
+  constructor(taskContainer, storage) {
+    this.#taskContainer = taskContainer;
+    this.#storage = storage;
+    this.storageAvailable = storage.storageAvailable();
+    if (this.storageAvailable) this.#activitiesArr = this.#storage.getTask();
+  }
+
+  get size() {
+    return this.#activitiesArr.length;
+  }
+
+  addTask(description) {
+    const id = this.size;
+    const object = this.#createTask(description, false, id);
+    this.#addToState(object);
+    const item = this.#createItem(object, id);
+    this.#taskContainer.append(item);
+  }
+
+  #saveLocal() {
+    if (this.storageAvailable) this.#storage.setTask(this.#activitiesArr);
+  }
+
+  #addToState(object) {
+    this.#activitiesArr.push(object);
+    this.#saveLocal();
+  }
+
+  #deleteFromState(index) {
+    this.#activitiesArr.splice(index, 1);
+    this.#saveLocal();
+    console.log(this.#activitiesArr);
```

Comment:
It's best not to leave any (console.logs)s in your code.

Reply:
Yeah i will prune all of them thank you dear reviewer!